server_addon/substancepainter/package.py

```diff
@@ -1,3 +1,11 @@
 name = "substancepainter"
 title = "Substance Painter"
-version = "0.1.1"
+version = "0.2.0"
+app_host_name = "substancepainter"
```

Comment:
My fault, it should be there, but `app_host_name` must be same for all addon versions. That changed in 1.1.1 server.
```suggestion
```